### PR TITLE
netcdf-c: add patch to enable '@4.9.0:~mpi+parallel-netcdf'

### PR DESCRIPTION
**Pull request by skosukhin**

### CODE REVIEW

**Review by tldahlgren — APPROVED** [review]
LGTM.

**Comment by tldahlgren** [comment]
Do you want me to wait until @WardF has a chance to comment before merging this PR?

**Comment by alalazo** [comment]
@tldahlgren For the sake of reducing pending PRs I'll be merging this, since @skosukhin is also a maintainer.

**Pull request merged**
